store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java

```diff
@@ -1129,7 +1129,7 @@ public boolean isEnableBatchPush() {
         return isEnableBatchPush;
     }
 
-    public void setEnableBatchPush(boolean enableBatchPush) {
+    public void setIsEnableBatchPush(boolean enableBatchPush) {
         isEnableBatchPush = enableBatchPush;
```

Comment:
IMO，change the isEnableBatchPush to enableBatchPush is better

Reply:
If change the var name I think it will break the compatibility.

Reply:
Now we can set it in broker.cfg by `enableBatchPush`, so use `enableBatchPush` directly is better. @absolute8511

Reply:
@cserwen since old isEnableBatchPush is ignored, and I think we can change the variable name `isEnableBatchPush` to `enableBatchPush`  without break compatibility.